### ISSUE TITLE
Cannot build the project

### ISSUE THREAD

**Issue by tr32** [zhongzhong0505/CodeBe-Admin#1]
Hello, i can not build the project, i get the following Error:

Cannot find module '@ angular/cdk/platform'
Cannot find module '@ angular/cdk/a11y'
Cannot find module '@ angular/cdk/bidi'

...
a lot of modules are not found
...

ERROR in C:/Dev/Design/CodeBe-Admin-master/src/app/login/login.module.ts (6,10): Module '"C:/Dev/Design/CodeBe-Admin-master/node_modules/@ angular/material/material"' has no exported member 'MdButtonModule'.

ERROR in C:/Dev/Design/CodeBe-Admin-master/src/app/login/login.module.ts (6,25): Module '"C:/Dev/Design/CodeBe-Admin-master/node_modules/@ angular/material/material"' has no exported member 'MdInputModule'.

ERROR in MdButtonModule is not an NgModule
